orchestrator.dockerfile

```diff
@@ -1,14 +1,14 @@
-FROM cogment/orchestrator:v1.0.3
+FROM cogment/orchestrator:v2.0.0
 
 RUN apt-get update && apt-get -y install gettext-base
 
 ADD cogment.yaml ./
 ADD *.proto ./
 
-ENV TRIAL_LIFECYCLE_PORT=9000
-ENV TRIAL_ACTOR_PORT=9000
+ENV COGMENT_LIFECYCLE_PORT=9000
+ENV COGMENT_ACTOR_PORT=9000
 #ENV PROMETHEUS_PORT=8000
 
 # Didn't manage to setup envsubst in the ENTRYPOINT
 ENTRYPOINT []
-CMD envsubst < /app/cogment.yaml > /app/cogment.yaml.out && orchestrator --config=/app/cogment.yaml.out
+CMD envsubst < /app/cogment.yaml > /app/cogment.yaml.out && orchestrator --params=/app/cogment.yaml.out
```

Comment:
Ok so from what I gather the param file shouldn't be used anymore, if it's not used we don't need to copy it and we don't need to `ensubst` it which probably mean we don't need a dockerfile for the orchestrator at all. Can you check that ?

Reply:
Can look thoroughly on Monday

Reply:
Dockerfile removed, you are right